src/app/shared/users.service.ts

```diff
@@ -1,33 +1,54 @@
 import { Injectable } from '@angular/core';
-import { usersList } from './users.mock';
 import { User } from './users.model';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
 
 @Injectable({
   providedIn: 'root'
 })
 export class UsersService {
 
-  constructor() { }
+  mainURL: string = 'http://localhost:8080/';
+  usersMainURL: string = `${this.mainURL}users/`;
 
-  getAllUsers(): User[] {
-    return usersList;
+  constructor(private http: HttpClient) { }
+
+  getAllUsers(): Observable<User[]> {
+    return this.http.get<User[]>(this.usersMainURL);
+  }
+
+  getActiveUsers(): Observable<User[]> {
+    return this.http.get<User[]>(this.usersMainURL).pipe(
+      map(usersList => usersList.filter(user => user.isDeleted === false))
```

Comment:
Instead of `user.isDeleted === false` you can write `!user.isDeleted`

Reply:
did this